src/v2/Components/ArtworkFilter/index.tsx

```diff
@@ -51,40 +46,42 @@ import {
 import { Sticky } from "v2/Components/Sticky"
 import { ScrollRefContext } from "./ArtworkFilters/useScrollContext"
 import { ArtworkSortFilter } from "./ArtworkFilters/ArtworkSortFilter"
-import { GeneArtworkFilter_gene } from "v2/__generated__/GeneArtworkFilter_gene.graphql"
 import type RelayModernEnvironment from "relay-runtime/lib/store/RelayModernEnvironment"
-import { TagArtworkFilter_tag } from "v2/__generated__/TagArtworkFilter_tag.graphql"
-import { Works_partner } from "v2/__generated__/Works_partner.graphql"
-import { CollectionArtworksFilter_collection } from "v2/__generated__/CollectionArtworksFilter_collection.graphql"
 import { ArtworkGridFilterPills } from "./SavedSearch/Components/ArtworkGridFilterPills"
 import { SavedSearchAttributes } from "./SavedSearch/types"
 import { extractPills } from "../SavedSearchAlert/Utils/extractPills"
 import { useFilterPillsContext } from "./SavedSearch/Utils/FilterPillsContext"
 import { getTotalSelectedFiltersCount } from "./Utils/getTotalSelectedFiltersCount"
 
+interface ArtworkFilterProps extends SharedArtworkFilterContextProps, BoxProps {
+  enableCreateAlert?: boolean
+  Filters?: JSX.Element
+  offset?: number
+  // Input variables passed to FilterArtworkConnection `input` argument
+  relayRefetchInputVariables?: object
+  // Root-level variables passed to Relay query
+  relayVariables?: object
```

Comment:
can't we use the relay-generated types for variables here instead of just object?

Reply:
We could, but that would then require us to pass in a union type here of all the various artwork filters that we're using throughout Force. Its not _so_ bad, but in practice it can get confusing since the filter is meant to be generic but is tied kinda lightly to the structure of the query. This is something that we can and should revisit when we refactor the Filter to be less dependent on GraphQL.